src/errors/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .base import BaseBotError
+from .http import GeneralHTTPError
```

Comment:
these errors are small enough to not warrant their own files. even disnake with tens of errors keeps them in a single file

Reply:
The base is just 1 file and then there is another file housing the errors, but yea I will make it 1 file in the main `src` directory

Reply:
Fixed.